test/testAbi.ts

```diff
@@ -1,16 +1,17 @@
 require('it-each')({ testPerIteration: true });
-const _ = require('lodash');
-const constants = require('./../src/constants');
-const abi = require('./../src/ethereumAbi');
-const randomWords = require('random-words');
-const crypto = require('crypto');
-const ethersAbi = require('@ethersproject/abi');
-const expect = require('chai').expect;
-const helpers = require('./testUtil/helpers');
-const question = require('readline-sync').question;
-const seedrandom = require('seedrandom');
-
-const encoder = new ethersAbi.AbiCoder;
+import { AbiCoder } from '@ethersproject/abi';
+import { expect as expect } from 'chai';
+import crypto from 'crypto';
+import _ from 'lodash';
+import randomWords from 'random-words';
+import { question as question } from 'readline-sync';
+import seedrandom from 'seedrandom';
+import { ETH_ABI_LATTICE_FW_TYPE_MAP, getFwVersionConst, HARDENED_OFFSET } from '../src/constants';
+import abi from './../src/ethereumAbi';
+import funcDef from './testUtil/etherscanABI_0xV2.json';
```

Comment:
Might be nice to just take the contents of this and replace `funcDef` in this test. We don't need this random JSON fine in the repo. No idea why I did it this way lol

Reply:
@douglance please also remove the JSON file from git

Reply:
Removed